cmd/statusd/main.go

```diff
@@ -252,19 +252,12 @@ func makeNodeConfig() (*params.NodeConfig, error) {
 	nodeConfig.LightEthConfig.Enabled = *lesEnabled
 	nodeConfig.SwarmConfig.Enabled = *swarmEnabled
 
-	if *standalone {
```

Comment:
Now, `standalone` is not used at all. I though we will keep it but only change its default value to `false`.

Reply:
yes, i created it before discussion